docs/kyma/docs/031-gs-local-installation.md

```diff
@@ -11,7 +11,7 @@ The document provides the prerequisites, the instructions on how to install Kyma
 
 ## Prerequisites
 
-To run Kyma locally, download these tools:
+To run Kyma locally, first clone this Git repository to your local machine and then download these tools:
```

Comment:
~first~ clone this Git repository to your local machine and ~then~ download these tools: